packages/office-addin-prettier-config/prettier.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    bracketSpacing: true,
+    printWidth: 100,
```

Comment:
Let's use the defaults. We want semi-colons and double-quotes. Trailing commas should use the default of none, otherwise it will put trailing commas in function call parameters which is weird. Bracket spacing defaults should also be good.

printWidth is the one option that I felt we may want to override to be 100 as 80 felt like it caused functions to wrap to having parameters on separate lines, which I personally don't like. I'd like to experiment to know more.

Reply:
I just looked at fabric-internal, and they use 120 for printWidth.

Reply:
https://github.com/OfficeDev/office-ui-fabric-react uses 140 for printWidth.

Reply:
Based on this, I think we should make printWidth to be 120 as a starting value.

Reply:
I went ahead and made these changes.

Reply:
@donaldgu Are you OK with these changes?